### PR TITLE
add `postcss-is-pseudo-class` to `postcss-preset-env` and `cli`

### PR DESCRIPTION
**Pull request by romainmenke**
_unsure is we should first promote the standalone plugin and wait for initial feedback?_

edit : decided to add this to other planned features for `postcss-preset-env` v7.3.0

### CODE REVIEW

File: plugin-packs/postcss-preset-env/src/lib/ids-by-execution-order.js [hunk 1]
```diff
@@ -12,7 +12,6 @@ export default [
 	'case-insensitive-attributes',
 	'focus-visible-pseudo-class',
 	'focus-within-pseudo-class',
-	'matches-pseudo-class', // run matches-pseudo-class and
```
Comment (romainmenke) [inline plugin-packs/postcss-preset-env/src/lib/ids-by-execution-order.js:15]: plugin was removed, still existed in the ordering

File: plugin-packs/postcss-preset-env/src/lib/ids-by-execution-order.js [hunk 2]
```diff
@@ -22,10 +21,10 @@ export default [
 	'hexadecimal-alpha-notation',
 	'lab-function',
 	'rebeccapurple-color',
-	'color-mod-function', // run color-mod after other color modifications have finished
```
Comment (romainmenke) [inline plugin-packs/postcss-preset-env/src/lib/ids-by-execution-order.js:25]: plugin was removed, still existed in the ordering

**Comment by Antonio-Laguna** [comment]
Is this ready to be reviewed & merged into 7.3?

**Comment by romainmenke** [comment]
Yes this is ready to be reviewed and merged :)

**Review by Antonio-Laguna — APPROVED** [review]
Nice one!

**Pull request merged**